test/contestant_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require "./lib/contestant"
+require "./lib/game"
+
+class ContestantTest < Minitest::Test
+
+  def setup
+    @alexander = Contestant.new({first_name: 'Alexander',                       last_name: 'Aigiades', age: 28,    state_of_residence: 'CO',
+    spending_money: 10})
+  end
+
+  def test_it_exists
+    assert_instance_of Contestant, @alexander
+  end
+
+  def test_contestant_attributes
+    assert_equal "Alexander Aigiades", @alexander.full_name
+    assert_equal 28, @alexander.age
+    assert_equal 'CO', @alexander.state_of_residence
+    assert_equal 10, @alexander.spending_money
+  end
+
+  def test_if_contestant_is_from_out_of_state
+    assert_equal false, @alexander.out_of_state?
```

Comment:
If a method should return either true or false, make sure you test both scenarios.